addons/website/views/website_templates.xml

```diff
@@ -2033,33 +2048,25 @@
 <template id="template_footer_call_to_action" inherit_id="website.layout" name="Call-to-Action" active="False">
     <xpath expr="//div[@id='footer']" position="replace">
         <div id="footer" class="oe_structure oe_structure_solo" t-ignore="true" t-if="not no_footer">
-            <section class="s_call_to_action pt64 pb64">
-                <div class="container">
-                    <div class="row">
-                        <div class="col-lg-9">
+            <section class="s_call_to_action s_text_block" data-snippet="s_text_block" data-name="Text">
```

Comment:
It seems it was already a mistake before to have call to action snippet without data-snippet but here you are adding two different snippet classes on the same element... that's not right. It probably should be a call to action snippet, not a text block, even if there is no real difference.